src/RolloutScorer/RolloutScorer/RolloutUploader.cs

```diff
@@ -72,88 +71,91 @@ public async static Task<int> UploadResultsAsync(List<string> scorecardFiles, Co
         /// <param name="githubClient">An authenticated Octokit.GitHubClient instance</param>
         /// <param name="storageAccountKey">Key to the rollout scorecards storage account</param>
         /// <param name="githubConfig">GitHubConfig object representing config</param>
-        public async static Task UploadResultsAsync(List<Scorecard> scorecards, GitHubClient githubClient, string storageAccountKey, GithubConfig githubConfig)
+        public async static Task UploadResultsAsync(List<Scorecard> scorecards, GitHubClient githubClient, string storageAccountKey, GithubConfig githubConfig, bool makePr = true)
```

Comment:
true being a default is... unexpected. :-) But probably fine.  Default parameters that aren't just "default" values, can lead to incorrect behaviors.

Personally I'd make it "skipPr" and invert the logic.

Reply:
ooh sensible. Will do that!